eqcorrscan/core/lag_calc.py

```diff
@@ -431,7 +433,7 @@ def _prepare_data(detect_data, detections, template, delays,
 def lag_calc(detections, detect_data, template_names, templates,
              shift_len=0.2, min_cc=0.4, horizontal_chans=['E', 'N', '1', '2'],
              vertical_chans=['Z'], cores=1, interpolate=False,
-             plot=False, parallel=True):
+             plot=False, plotdir='.', parallel=True):
```

Comment:
I think the default should be `None` and if plotdir is None then the figure should be shown, which is currently the expected behaviour. See the comment below about how plotdir could be used.

Reply:
Yes. I prefer this method too. But i want to make it look like match_filter.